src/test/java/com/google/sps/StreamServletTest.java

```diff
@@ -65,15 +65,13 @@ public void testDoGet() throws Exception {
     servlet.doGet(mockRequest, mockResponse);
     verify(mockResponse, atLeast(1)).setContentType("application/json");
 
-    // Convert JSON File to Java Object
+    //Convert Java Object to Json
     Gson gson = new Gson();
-    Stream testStream = gson.fromJson(stringWriter.toString(), Stream.class);
+    Stream stream = new Stream();
+    String actualStream = gson.toJson(stream).trim();
+    //stringWriter adds an extra space
```

Comment:
Add a space after //. Also say We're trimming the strings since the Writer classes can add extra spaces in the string.